content/api/commands/hover.md

```diff
@@ -80,13 +80,12 @@ cy.get('.checkbox').check({ force: true })
 
 ### Add as a custom command
 
-Although Cypress does not have a built-in **cy.hover()**, it is a reserved
-command. Therefore, to add `cy.hover()` as a custom command;
-`Cypress.Commands.overwrite()` should be used instead of
-`Cypress.Commands.add()`.
+Although Cypress does not have a built-in `cy.hover()`, you can add your own
+custom command using
```

Comment:
Can you add a note on this page saying that while `Cypress.Commands.overwrite()` still works, as of Cypress 10, we recommend using `Cypress.Commands.add()`.